about.html

```diff
@@ -0,0 +1,250 @@
+          <a href="#" class="top-navbar-text">English</a>
+          <a href="#" class="top-navbar-text">My Page</a>
+          <a href="#" class="top-navbar-text">Logout</a>
+        </div>
+      </div>
+
+      <div class="second-navbar">
+
+          <a href="index.html" class="brand">
+            <h2 class="logo">Porsche</h2>
+          </a>
+
+          <nav class="navbar">
+
+            <ul class="nav-list">
+
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#program" class="nav-link">Program</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#join" class="nav-link">Join</a>
+              </li>
+              
+              <li class="nav-item">
+                <a href="#sponsor" class="nav-link">Sponsor</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#news" class="nav-link">News</a>
+              </li>
+              
+              <li class="nav-item">
+                <a href="#" class="nav-link-cta">Porsche Cars</a>
+              </li>
+
+            </ul>
+
+          </nav>
+
+          <div class="icons">
+            <div class="fas fa-bars" id="menu-btn"></div>
+          </div>
+
+      </div>
+
+    </header>
+
+  </section>
+
+  <!-- HERO SECTION STARTS -->
+
+
+  <section class="hero-about">
+    <div class="hero-content-about">
+      <h5 class="hero-heading-about-subtitle">Class Is Permanent</h5>
+      <h2 class="hero-heading-about-title">We are the Porsche Club of America</h2>
+      <p class="hero-para-about">Porsche Cars North America, Inc. (PCNA), based in Atlanta, Georgia, is the exclusive
+        importer of Porsche 911, 718 Boxster and 718 Cayman sports cars
+        and the Cayenne, Macan and Panamera sports activity vehicles.
+      </p>
+
+      <p class="hero-contact-text-about">
+        Please contact us per email porsche@porsche.com or via <a href="#">1-800-767-7243</a> or
+        visit our website at <i>www.porsche.com</i> to locate a Porsche dealer near you.
+      </p>
+    </div>
+  </section>
+
+  <!-- HERO SECTION ENDS -->
+
+  <!-- INFORMATION SECTION STARTS -->
+
+  <section class="information">
+    <div class="info-content">
+      <h2 class="info-heading">Porsche Cars North America</h2>
+      <div class="info-underline"></div>
+      <p class="info-para">The Logo for our next model was decided through the logo competition from November 24 - to
+        December 11.
+      </p>
+
+      <p class="info-catchy-text">
+        PORSCHE AG the SUVs DEALERS.
+      </p>
+    </div>
+
+    <div class="hr-divider"></div>
+  </section>
+
+  <!-- INFORMATION SECTION STARTS -->
+
+  <!-- LATEST NEWS SECTION STARTS -->
+
+  <section class="latest-news">
+    <div class="latest-news-content">
+
+      <h2 class="latest-news-heading">See Latest 2022 Porsche Models</h2>
```

Comment:
- [ ] Kindly reduce the space between the logo and pas events section 👍 

![image](https://user-images.githubusercontent.com/50721479/203853252-eb944e99-20f9-40e0-b9f2-8c317df367d8.png)